aten/src/ATen/native/Embedding.cpp

```diff
@@ -14,6 +14,22 @@
 
 namespace at { namespace native {
 
+Tensor embedding_dense_double_backward(
```

Comment:
I think you want to move this to tools/autograd/templates/Functions.cpp.

Reply:
In general, only functions to be called directly by the user or those needing an explicit derivative are included in ATen. If you look at Functions.cpp, you see that it has derivatives for anything and everything in there. (I'll happily admit that when I put my first function there, it seemed very strange to me, too, but you'll get used to it over time, too.)

Reply:
Oh , now it does make sense to move it their.